src/Relation/Binary/Bundles.agda

```diff
@@ -299,6 +299,23 @@ record StrictTotalOrder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) wh
   Please use Eq.decSetoid instead."
   #-}
 
+record DenseLinearOrder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
+  infix 4 _≈_ _<_
+  field
+    Carrier            : Set c
+    _≈_                : Rel Carrier ℓ₁
+    _<_                : Rel Carrier ℓ₂
+    isDenseLinearOrder : IsDenseLinearOrder _≈_ _<_
+
+  open IsDenseLinearOrder isDenseLinearOrder public
+
+  strictTotalOrder : StrictTotalOrder c ℓ₁ ℓ₂
+  strictTotalOrder = record
+    { isStrictTotalOrder = isStrictTotalOrder
+    }
+
+  --open StrictTotalOrder strictTotalOrder public
```

Comment:
So to resolve this problem, you have to open first thing A (either the structure or the bundle), and then open the other one exporting only the things missing in A. 

Here would recommend opening `StrictTotalOrder` as you'll get all the bundles and structures, and only have to import `dense` from `IsStrictTotalOrder`, i.e. `open IsDenseLinearOrder isDenseLinearOrder using (dense)`

Reply:
... and , in fact, `using (isStrictTotalOrder; dense)` in order to be able to create the `StrictTotalOrder` sub-bundle in the first place.